package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-server-sdk",
-  "version": "3.10.0",
```

Comment:
In general, avoid bumping versions in PRs. Package maintainers will bump the versions before publishing.

Reply:
OK.

Reply:
I just just deleted the commit.